wiki/Tournaments/SS_Tournament/2018/en.md

```diff
@@ -0,0 +1,205 @@
+# SS Tournament 2018
+
+![SS Tournament 2018](Banner.jpg)
+
+SS Tournament 2018 is a osu!standard tournament organized and ran by ![][flag_CA] [Sparhten](https://osu.ppy.sh/users/7601720), ![flag_US] [NeonLights](https://osu.ppy.sh/users/5277199), and ![] [flag_GR] [FrozenPheonix](https://osu.ppy.sh/users/9696149).
```

Comment:
omit space in-between `]` and `(` (it happens twice in this paragraph)

Reply:
i dont see it?

Reply:
Sorry about that, I noticed the mistake, but wrote the wrong response. There are two issues here:

- missing `[]` between `!` and `[flag_US]` (should be `![][flag_US]`)
- omit the space between `![]` and `[flag_GR]` (should be `![][flag_GR]`)

`SS Tournament 2018 is a osu!standard tournament organized and ran by ![][flag_CA] [Sparhten](https://osu.ppy.sh/users/7601720), ![][flag_US] [NeonLights](https://osu.ppy.sh/users/5277199), and ![][flag_GR] [FrozenPheonix](https://osu.ppy.sh/users/9696149).`